src/DropDownMenu/DropDownMenu.js

```diff
@@ -111,6 +111,10 @@ class DropDownMenu extends Component {
      */
     disabled: PropTypes.bool,
     /**
+     * Overrides default icon component
+     */
+    iconButtonElement: PropTypes.element,
```

Comment:
I wouldn't follow that example. Let's follow what we are doing in the `next` branch. `node` would be better.